gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/task/KubernetesConfigViewTaskTest.java

```diff
@@ -44,13 +57,14 @@ public void setUp() throws IOException {
   @Test
   public void runTask_withManualSettings_shouldLogThem() {
     // Given
+    doNothing().when(taskEnvironment.logger).lifecycle(loggedMessages.capture());
     extension.buildStrategy = JKubeBuildStrategy.s2i;
     extension.isOffline = true;
     final KubernetesConfigViewTask configViewTask = new KubernetesConfigViewTask(KubernetesExtension.class);
     // When
     configViewTask.runTask();
     // Then
-    verify(taskEnvironment.logger, times(1))
-        .lifecycle(matches("k8s: \n---\noffline: true\nbuildStrategy: \"s2i\""));
```

Comment:
Wouldn't it be more simpler to replace with a valid regex?
Something like:
```
.lifecycle(matches("k8s: \r?\n---\r?\noffline: true\r?\nbuildStrategy: \"s2i\""));
```

Reply:
If I remember correctly, I adjusted this in the code generating these logs as well, because one line was using the `String.format` with `%n` in the string to print, but the other lines were using an hardcoded `\n`.
I can, of course, use such regex here, but I think that the logs should be consistent upfront (so, in the code which generates them).
Moreover, the logger gets called with other messages as well, in fact I'm quite surprised this test passed even on the CI in the past.

Reply:
This regex should work regardless of the inconsistency of line breaks.
Of course, the inconsistency source should also be changed.

> Moreover, the logger gets called with other messages as well, in fact I'm quite surprised this test passed even on the CI in the past.

Yes, but the verify matcher checks that the method was called exactly once with the provided regex, despite any of the other recorded calls.

Reply:
Yep, but the error I was getting is that it was expecting that string but it got a completely different one. 
Looking at the code, seems like you log some other information before that line and it was trying to match the passed string again those information.
I'll give it again a try and report back to you the error in case the regex doesn't help.